tests/unit/plugins/modules/packaging/os/test_rhsm_release.py

```diff
@@ -125,12 +125,10 @@ def test_release_insane(self):
 
     def test_release_matcher(self):
         # throw a few values at the release matcher -- only sane_values should match
-        sane_values = ['1Server', '10Server', '1.10', '10.0']
+        sane_values = ['1Client', '10Server', '1.10', '10.0', '9']
```

Comment:
What is the reason of removing existing tests? How about this?
```suggestion
        sane_values = ['1Server', '1Client', '10Server', '1.10', '10.0', '9']
```

Reply:
@aminvakil Initially I think replace '1Server' as '1Client' would not decrease the test coverage. It is OK for me to keep the '1Server' test case and I submit the change.
Looks like the third-party check fails in one ssh connection which should not caused by my code change.

Reply:
> @aminvakil Initially I think replace '1Server' as '1Client' would not decrease the test coverage. It is OK for me to keep the '1Server' test case and I submit the change.

If someone removes `Server` from code and `1Server` is not in test cases, it wouldn't fail, right? Thanks for adding that again.

> Looks like the third-party check fails in one ssh connection which should not caused by my code change.
Yes, you're right. This is failing on other PRs as well, it's unrelated to your PR.